### DISCUSSION TITLE
Albumentations compatibility with obb format

### DISCUSSION THREAD

**Discussion by jcantero-equip** [albumentations-team/albumentations#2170] [Q&A]
I have been looking into it through the documentation and also into the discussions, but I haven't found any mention to the obb format. Are you planning to add this format to the Albumentations bbox augmentation?

**Comment by ternaus** [comment]
I did not think about obb format yet. It is possible to add it, will add as feature request, but cannot name any timeline for this feature.

**Comment by jaffe-fly** [comment]
also need `yolo obb` format,   hope can add it soon， or  how to add it by custom

**Comment by ternaus** [comment]
@jaffe-fly 

When monthly donations, will reach $5000, will hire a person to work full time on the library.

The first task this person will get - add obb format.

If you would like to speed up this moment, just become a sponsor: https://github.com/sponsors/albumentations-team
